plugins/forum/lib/foodsoft_forum/engine.rb

```diff
@@ -0,0 +1,20 @@
+module FoodsoftForum
+  class Engine < ::Rails::Engine
+    def navigation(primary, context)
+      return unless FoodsoftForum.enabled?
+      primary.item :forum, I18n.t('navigation.forum'), '/f/forums', id: nil do |subnav|
+        for category in  Forem::Category.all
+          subnav.item :forem_category, category.name, '/f/forums/categories/' + category.name.downcase, id: nil
+        end
+      end
```

Comment:
Found it! `context.forem.category_path(category)`

You can replace `'/f/forums'` by `'#'` since it's a parent menu item. Or, if you really want it, use `context.forem_path`.